history-of-quakers/modernized/19-chapter-17.adoc

```diff
@@ -549,6 +571,9 @@ wherefore he named this child Abigail, signifying, the father`'s joy.
 In this testament he gave her instructions for the future conduct of her life,
 the beginning whereof is thus:
 
+[.embedded-content-document.letter]
```

Comment:
Not sure if this should be called a letter....it's called an epistle in the paragraph above it, but it's only to one person. It's also called a testament, so maybe it should be labeled a testimony, but didn't sound so testimony-ish to me....more instructional. Idk.

Reply:
letter seems good. i love this letter. did you read it? i just read a big chunk of this to my family a few days ago. it's sooo good. ❤️

Reply:
I didn't, but I will!!